benches/benches/db_lookup_times.rs

```diff
@@ -0,0 +1,88 @@
+use crate::db_lookup_times_utils::{
+    matrix::matrix,
+    utils::{
+        get_full_block,
+        get_random_block_height,
+        multi_get_block,
+        open_db,
+        open_raw_rocksdb,
+    },
+};
+use criterion::{
+    criterion_group,
+    criterion_main,
+    Criterion,
+};
+use db_lookup_times_utils::seed::{
+    seed_compressed_blocks_and_transactions_matrix,
+    seed_full_block_matrix,
+};
+use fuel_core_storage::transactional::AtomicView;
+
+mod db_lookup_times_utils;
+
+pub fn header_and_tx_lookup(c: &mut Criterion) {
+    let method = "header_and_tx";
+
+    seed_compressed_blocks_and_transactions_matrix(method);
+    let mut group = c.benchmark_group(method);
+
+    for (block_count, tx_count) in matrix() {
+        let database = open_db(block_count, tx_count, method);
+        let height = get_random_block_height(block_count);
+        group.bench_function(format!("{block_count}/{tx_count}"), |b| {
+            b.iter(|| {
+                let view = database.latest_view().unwrap();
+                let _ = view.get_full_block(&height);
+            });
+        });
+    }
+
+    group.finish();
+}
+
+pub fn multi_get_lookup(c: &mut Criterion) {
+    let method = "multi_get";
+
+    seed_compressed_blocks_and_transactions_matrix(method);
+    let mut group = c.benchmark_group(method);
+
+    for (block_count, tx_count) in matrix() {
+        let database = open_raw_rocksdb(block_count, tx_count, method);
+        let height = get_random_block_height(block_count);
+        group.bench_function(format!("{block_count}/{tx_count}"), |b| {
+            b.iter(|| {
+                multi_get_block(&database, height);
+            });
+        });
+    }
+
+    group.finish();
+}
+
+pub fn full_block_lookup(c: &mut Criterion) {
+    let method = "full_block";
+
+    seed_full_block_matrix();
+    let mut group = c.benchmark_group(method);
+
+    for (block_count, tx_count) in matrix() {
+        let database = open_db(block_count, tx_count, method);
+        let height = get_random_block_height(block_count);
+        group.bench_function(format!("{block_count}/{tx_count}"), |b| {
+            b.iter(|| {
+                let view = database.latest_view().unwrap();
```

Comment:
I think it is better to create view before the benchmark=)

Reply:
its draft, im still cleaning up !

Reply:
addressed!